server/src/test/java/org/elasticsearch/index/engine/LiveVersionMapTests.java

```diff
@@ -137,24 +142,25 @@ public void testConcurrently() throws IOException, InterruptedException {
                             }
                             if (isDelete == false && rarely()) {
                                 versionValue = new DeleteVersionValue(versionValue.version + 1, versionValue.seqNo + 1,
-                                    versionValue.term, Long.MAX_VALUE);
+                                    versionValue.term, clock.incrementAndGet());
```

Comment:
I can comment on  two lines above, so comment here: `map.removeTombstoneUnderLock(bytesRef);` isn't meant for public usage, right? if we index on a delete, the delete should go away.

Reply:
it's not meant for public use